parsl/executors/high_throughput/process_worker_pool.py

```diff
@@ -753,8 +753,8 @@ def manager_is_alive():
             worker_enqueued = True
 
         try:
-            # The worker will receive {'task_id':<tid>, 'buffer':<buf>}
             req = task_queue.get(timeout=task_queue_timeout)
+            # req is {'task_id':<tid>, 'buffer':<buf>, 'context':<dict>}
```

Comment:
in a few other places i've pushed towards TypedDict for this. I don't really know how much enforcement you get on the receiving end (none?) but it's at least something on the sending side.

Reply:
i don't have a problem with this in this PR, it a passing comment on the theme of "never write comments as `# comments`".

Reply:
Heh.  Believe it or not, that comment was a straw on the motivation-camel for my thought-process on the MessagePack route.